lib/src/shops/shop_details_screen.dart

```diff
@@ -8,7 +8,7 @@ import 'package:provider/provider.dart';
 import 'models/shop_model.dart';
 
 class ShopDetailScreen extends StatelessWidget {
-  static String routeName = '/shop-detail';
+  static const String routeName = '/shop-detail';
```

Comment:
You think we should have a file with all of this navigation constants, instead of being inside each class? Just a food for thought

Reply:
I like both approaches and I think they both scale well though I prefer keeping everything related to any particular widget within the same class just to reduce the amount of effort necessary to understand where everything is. With that said, I'm not opposed to using a global constants class. Let's just document whichever convention we go for so that any future team members can easily use it.